packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -1412,6 +1412,87 @@ describe('TableView', function () {
         moveFocus('S');
         expect(document.activeElement).toBe(getCell(tree, 'Sam'));
       });
+
+      describe('', function () {
```

Comment:
Mind updating this describe and the test case title so it is specific to this fix?

Reply:
I didn't want to add a new story because it's covered pretty well by the test and it's a big enough example to be annoying
let me see if I can do the add it as a story and use it in the test

Reply:
done